packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -967,6 +968,10 @@ class Scaffold extends StatefulWidget {
   /// Typically a [Drawer].
   final Widget endDrawer;
 
+  /// The color to use for the scrim that obscures primary content while a drawer is open.
+  /// By default, the color is [Colors.black54]
```

Comment:
Same comment about a line break between the first sentence and the rest.

Reply:
done